Sources/Apodini/SyntaxTreeVisitor/SyntaxTreeVisitor.swift

```diff
@@ -19,13 +19,14 @@ enum Scope {
 /// The `SyntaxTreeVisitable` makes a type discoverable by a `SyntaxTreeVisitor`.
 ///
 /// Each `Component` that needs to provide a custom `accept` implementation **must** conform to `SyntaxTreeVisitable` and **must** provide a custom `accept` implementation.
-protocol SyntaxTreeVisitable {
+public protocol SyntaxTreeVisitable {
+    /// Accept a visiting SyntaxTreeVisitor
     func accept(_ visitor: SyntaxTreeVisitor)
 }
 
 
 /// The `SyntaxTreeVisitor` is used to parse the Apodini DSL and forward the parsed result to the `SemanticModelBuilder`s.
-class SyntaxTreeVisitor {
+public class SyntaxTreeVisitor {
```

Comment:
Why do we need to make `SyntaxTreeVisitor` public? I would consider that be Apodini internals.

Reply:
Its because of this extension in: GRPCMethodModifier.swift

```
extension GRPCMethodModifier: SyntaxTreeVisitable {
    public func accept(_ visitor: SyntaxTreeVisitor) {
        visitor.addContext(GRPCMethodNameContextKey.self, value: methodName, scope: .current)
        component.accept(visitor)
    }
}
```